metadata-system.json

```diff
@@ -2,19 +2,27 @@
   "msgs":{
     "nl":{
       "configure.cookies.notice.message": "Gelieve je browser te configureren zodat cookies van derden worden geaccepteerd. Je kan meer lezen over ons cookie-beleid in onze voorwaarden.",
-      "cookie.notice.message": "Deze website gebruikt cookies. Door deze website te gebruiken zonder je instellingen te wijzigen ga je akkoord met onze voorwaarden."
+      "cookie.notice.message": "Wij respecteren jouw privacy. Er wordt enkel gebruik gemaakt van Google analytics paginaweergaven en acties om jou de beste ervaring te geven op deze website.",
```

Comment:
Ik zou "Google Analytics" niet expliciet vermelden. Stel dat het Google Tag Manager is bv, dan klopt het al niet meer. Liever een meer generieke boodschap als bv "Onze partners verzamelen statistische gegevens van het gebruik van de website om jou de meest optimale surfervaring te geven."